include/Makefile-comment

```diff
@@ -3,49 +3,51 @@ SHELL=/bin/bash
 DIRNAME=$(shell basename "$(PWD)")
 META=meta.ily
 LYS=score.ly voices.ily $(META) lyrics.ily output.ily ../../../include/header.ily ../../../include/paper.ily ../../../include/sound.ily ../../../include/version.ily
+
 NSUSF := $(shell git rev-parse --show-toplevel)
 VOLUMESUFFIX := $(if $(and $(shell command -v getenforce),$(findstring Enforcing,$(shell getenforce))),:Z,)
+DOCKERRUN := docker run -e LANG=C.UTF-8 -e LC_ALL=C.UTF-8 -v "$(NSUSF):$(NSUSF)$(VOLUMESUFFIX)" -w "$(PWD)" -i -t --rm
+
 LILYHOST := $(shell command -v lilypond)
-LILYDOCKER := docker run -e LANG=C.UTF-8 -e LC_ALL=C.UTF-8 -v "$(NSUSF):$(NSUSF)$(VOLUMESUFFIX)" -w "$(PWD)" -i -t --rm docker.io/nunsingetundseidfroh/lilypond lilypond
+LILYDOCKER := $(DOCKERRUN) docker.io/nunsingetundseidfroh/lilypond lilypond
 LILY := $(or $(LILYHOST),$(LILYDOCKER))
+
 XELAHOST := $(shell command -v xelatex)
-XELADOCKER := docker run -e LANG=C.UTF-8 -e LC_ALL=C.UTF-8 -v "$(NSUSF):$(NSUSF)$(VOLUMESUFFIX)" -w "$(PWD)" -i -t --rm docker.io/nunsingetundseidfroh/lilypond xelatex
+XELADOCKER := $(DOCKERRUN) docker.io/nunsingetundseidfroh/xelatex xelatex
 XELA := $(or $(XELAHOST),$(XELADOCKER))
 
-all: combined midi zip 
+PYTHONHOST := $(shell command -v python3)
+PYTHONDOCKER := $(DOCKERRUN) docker.io/python:3 python
+PYTHON := $(or $(PYTHONHOST),$(PYTHONDOCKER))
 
-combined: commentary sheet
+GSHOST := $(shell command -v gs)
+GSDOCKER := $(DOCKERRUN) docker.io/minidocks/ghostscript:latest
+GS := $(or $(GSHOST),$(GSDOCKER))
 
-sheet: $(DIRNAME)-sheet.pdf
+all: midi zip 
 
-commentary: translation $(DIRNAME)-commentary.pdf
+sheet: $(DIRNAME)-sheet.pdf
 
 midi: $(DIRNAME).midi
 
 zip: $(DIRNAME).zip
 
-combined: sheet commentary
-	gs -dBATCH -dNOPAUSE -q -sDEVICE=pdfwrite -dDOPDFMARKS=true -sOutputFile="$(DIRNAME).pdf" "$(DIRNAME)-sheet.pdf" pdfmarks "$(DIRNAME)-commentary.pdf"
-	# rm "$(DIRNAME)-sheet.pdf"
-	# rm "$(DIRNAME)-commentary.pdf"
-
-translation: $(wildcard translation.ilf)
-ifeq (,"$(wildcard translation.ilf)")
-	python3 ../../../tools/ilf2iltex.py translation.ilf
-endif
+$(DIRNAME).pdf: $(DIRNAME)-sheet.pdf $(DIRNAME)-commentary.pdf
+	$(GS) -dBATCH -dNOPAUSE -q -sDEVICE=pdfwrite -dDOPDFMARKS=true -sOutputFile="$(DIRNAME).pdf" "$(DIRNAME)-sheet.pdf" "$(DIRNAME)-commentary.pdf"
 
 $(DIRNAME)-sheet.pdf $(DIRNAME).midi: $(LYS)
 	$(LILY) -dno-point-and-click -o "$(DIRNAME)" $<
-	gs -dBATCH -dNOPAUSE -sDEVICE=pdfwrite -sOutputFile="$(DIRNAME)"-sheet.pdf "$(DIRNAME)".pdf
-	# mv "$(DIRNAME)".pdf "$(DIRNAME)-sheet.pdf"
+	mv "$(DIRNAME).pdf" "$(DIRNAME)-sheet.pdf"
+
+translation.iltex: translation.ilf
+	$(PYTHON) ../../../tools/ilf2iltex.py translation.ilf
 
-$(DIRNAME)-commentary.pdf:
+$(DIRNAME)-commentary.pdf: commentary.tex translation.iltex
```

Comment:
Commentaries do not necessarily have a translation. So, I think, this dependency works for Virga Jesse, but might fail for future commentaries, which do not happen to have a transaltion.ilf

Reply:
fixed via last example from http://kirste.userpage.fu-berlin.de/chemnet/use/info/make/make_4.html#SEC24